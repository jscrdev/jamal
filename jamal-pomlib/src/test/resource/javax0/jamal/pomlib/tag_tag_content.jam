{@import res:javax0/jamal/pomlib/pom.jim}{@comment

This file tests that a tag, defined in the pom.ji using the tag macro can be used empty, in which case the content will
be inserted as after the macro and an [end tag] should be used at the end or with content.

}{packaging}jar{end packaging}
{packaging jar}