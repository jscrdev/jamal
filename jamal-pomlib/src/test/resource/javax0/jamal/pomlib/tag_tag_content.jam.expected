<packaging>jar</packaging>
<packaging>jar</packaging>